examples/02_decoding/plot_miyawaki_encoding.py

```diff
@@ -0,0 +1,233 @@
+"""
+============================================================
+Encoding models for visual stimuli from Miyawaki et al. 2008
+============================================================
+
+This example demonstrates how to build an encoding model for
+functional MRI data with visual stimuli.
+
+Encoding models try to predict neuronal activity using properties of
+a real world stimulus, like images or sound.
+
+In the dataset from :func:`nilearn.datasets.fetch_miyawaki2008`,
+participants were shown images consisting of 10x10
+binary (either black or white) pixels and the corresponding  fMRI activity
+was recorded. We will predict the neuronal activity from the pixel-values
+of the presented images for each voxel. Then we extract the receptive fields
+for a set of voxels to see which pixel location a voxel is most sensitive to.
+
+See also :doc:`plot_miyawaki_reconstruction` for a decoding
+approach for the same dataset.
+"""
+
+##############################################################################
+# Loading the data
+# ----------------
+# First some imports:
+
+import numpy as np
+import pylab as plt
+from nilearn.plotting import plot_stat_map
+from nilearn import masking
+
+
+##############################################################################
+# Now we can load the data set:
+from nilearn.datasets import fetch_miyawaki2008
+
+dataset = fetch_miyawaki2008()
+
+##############################################################################
+# We only use the training data of this study,
+# where random binary images were shown.
+
+X_random_filenames = dataset.func[12:]
+y_random_filenames = dataset.label[12:]
+
+# shape of the binary image in pixels
+y_shape = (10, 10)
```

Comment:
Maybe we could plot one or two image here, and have a text reminding us that these were the stimuli.